Sources/MapboxNavigation/PreviewOptions.swift

```diff
@@ -0,0 +1,12 @@
+import MapboxCoreNavigation
+
+// :nodoc:
```

Comment:
Do we intend to keep all these new types undocumented once the feature fully lands? The main reason we used to sprinkle `:nodoc:` everywhere was that we needed to expose various UI classes to UIAppearance for styling but otherwise wouldn’t have wanted them to be public. Is that the case for these types too?

Reply:
Most of the publicly exposed symbols will be documented in scope of different PR. I still plan to leave some public symbols undocumented because I'm not 100% sure whether they're really something that we'll need.